src/components/menus/item.js

```diff
@@ -12,6 +12,18 @@ export const PanelRowContainer = styled(Flex)`
   ${props => props.selected && `background: ${getColor("selected")(props)};`}
 `
 
+const StyledIcon = styled(Icon)`
```

Comment:
I think `Icon` accepts `height` and `width` props, so maybe could be added as `attrs`! 😄